docs/mutect/mutect.tex

```diff
@@ -236,6 +236,26 @@ \section{Germline Filter}\label{germline-filter}
 
 To filter, we set a threshold on this posterior probability.
 
+\section{Contamination Filter}\label{contamination-filter}
+Suppose our tumor bam has contamination fraction $\alpha$ and that at some site we have $a$ alt reads out of $d$ total reads.  Suppose further that the alt allele has population allele frequency $f$.  We will compute a simple estimate of the posterior probability that these alt reads came from a contaminating sample and not from a true somatic variant.  Letting $\pi$ be the prior probability of somatic variation as above a very crude model for the alt count distribution of somatic variation is a uniform distribution.  That is, we assume that any value of $a$ from $0$ to $d$ is equally likely.  Then the likelihood of the data given a true somatic variant is
+\begin{equation}
+P(a | {\rm somatic}) = \frac{\pi}{d + 1}.
+\end{equation}
+
+We consider two models of contamination.  If there are multiple contaminants we approximate each contaminant read as independent.  Then the probability of any given read being an alt contaminant read is $\alpha f$, so we have
+\begin{equation}
+P(a | {\rm many~contaminant}) = {\rm Binom}(a | d, \alpha f).
+\end{equation}
+If there is a single contaminating sample it is heterozygous with probability $2f(1-f)$ and homozygous for the alt with probability $f^2$, in which cases fractions $\alpha/2$ and $\alpha$ of all reads to be alt contaminants.  Thus
+\begin{equation}
+P(a | {\rm one~contaminant}) = 2f(1-f) {\rm Binom}(a | d, \alpha /2) + f^2 {\rm Binom}(a | d, \alpha).
```

Comment:
`P(a | one contaminant)` isn't normalized as is. I think adding `(1-f)^2 I(a =0)`, where I() is the indicator, will fix it (or did you omit it on purpose?)

Reply:
Laziness, fixed.